cmd/ike/cmd/develop.go

```diff
@@ -0,0 +1,106 @@
+package cmd
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"strconv"
+	"strings"
+	"sync"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	deploymentName string
+	port           int
+	runnable       string
+	method         string
+)
+
+func init() {
+	developCmd.PersistentFlags().StringVarP(&deploymentName, "deployment", "d", "", "name of the deployment or deployment config")
+	developCmd.PersistentFlags().IntVarP(&port, "port", "p", 8000, "port to be exposed")
+	developCmd.PersistentFlags().StringVarP(&runnable, "run", "r", "", "command to run your application")
+	developCmd.PersistentFlags().StringVarP(&method, "method", "m", "inject-tcp", "telepresence proxying mode - see https://www.telepresence.io/reference/methods")
+
+	_ = developCmd.MarkPersistentFlagRequired("deployment")
+	_ = developCmd.MarkPersistentFlagRequired("run")
+	rootCmd.AddCommand(developCmd)
+}
+
+const telepresenceBin = "telepresence"
+
+var developCmd = &cobra.Command{
+	Use:   "develop",
+	Short: "starts the development flow",
+	Long:  `TODO`,
+	Run: func(cmd *cobra.Command, args []string) {
+
+		checkIfTelepresenceExists()
```

Comment:
Shouldn't this 'stop' the further execution?

Reply:
Right, with `logrus` it did without explicit `exit` call. Added in 9f08970